includes/specials/SpecialBlockList.php

```diff
@@ -324,7 +326,7 @@ function formatValue( $name, $value ) {
 					$formatted = Linker::userLink( $value, $row->by_user_name );
 					$formatted .= Linker::userToolLinks( $value, $row->by_user_name );
 				} else {
-					$formatted = htmlspecialchars( \User::getUsername( $row->ipb_by, $row->ipb_by_text ) ); // foreign user?
+					$formatted = htmlspecialchars( $row->ipb_by_text ); // foreign user?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)